pkg/strategy/supertrend/strategy.go

```diff
@@ -70,21 +119,33 @@ type Strategy struct {
 	// SupertrendMultiplier ATR multiplier for calculation of supertrend
 	SupertrendMultiplier float64 `json:"supertrendMultiplier"`
 
+	// LinearRegression Use linear regression as trend confirmation
+	LinearRegression *LinGre `json:"linearRegression,omitempty"`
+
 	// Leverage
 	Leverage float64 `json:"leverage"`
 
-	// TakeProfitMultiplier TP according to ATR multiple, 0 to disable this
-	TakeProfitMultiplier float64 `json:"takeProfitMultiplier"`
+	// TakeProfitAtrMultiplier TP according to ATR multiple, 0 to disable this
+	TakeProfitAtrMultiplier float64 `json:"takeProfitAtrMultiplier"`
 
-	// StopLossByTriggeringK Set SL price to the low of the triggering Kline
+	// StopLossByTriggeringK Set SL price to the low/high of the triggering Kline
 	StopLossByTriggeringK bool `json:"stopLossByTriggeringK"`
 
-	// TPSLBySignal TP/SL by reversed signals
-	TPSLBySignal bool `json:"tpslBySignal"`
+	// StopByReversedSupertrend TP/SL by reversed supertrend signal
+	StopByReversedSupertrend bool `json:"stopByReversedSupertrend"`
+
+	// StopByReversedDema TP/SL by reversed DEMA signal
+	StopByReversedDema bool `json:"stopByReversedDema"`
+
+	// StopByReversedLinGre TP/SL by reversed linear regression signal
+	StopByReversedLinGre bool `json:"stopByReversedLinGre"`
 
 	currentTakeProfitPrice fixedpoint.Value
 	currentStopLossPrice   fixedpoint.Value
 
+	// ExitMethods Exit methods
+	ExitMethods []bbgo.ExitMethod `json:"exits"`
```

Comment:
use bbgo.ExitMethodSet

Reply:
ExitMethodSet is not in this branch yet
replace it later?